src/ui/x11/x11_ui.cpp

```diff
@@ -52,8 +52,42 @@ void UiWindowX11::_display()
             for ( size_t i = 0u; i < _point.size(); ++i ) {
                 const Point2d & point = _point[i].first;
                 XSetForeground( _uiDisplay, defaultGC, _point[i].second );
-                XDrawLine( _uiDisplay, _window, defaultGC, static_cast<int>(point.x - 1), static_cast<int>(point.y - 1),
-                           static_cast<int>(point.x + 1), static_cast<int>(point.y + 1) );
+                XDrawLine( _uiDisplay, _window, defaultGC, static_cast<int>( point.x - 1 ), static_cast<int>( point.y - 1 ), static_cast<int>( point.x + 1 ),
```

Comment:
Hi @fullset could you please revert these changes as they are not relevant to this issue?